gitextensions/gitextensions
# Incorrect references and dependencies

Corrects #4269

    GitExtensions (cs project) has References to the plugin .dll which should not occur (no direct access to the .dll)
    Build dependencies do not include all subprojects. So for instance when debugging, a clean was required first (partly hidden by the previous issue).

How did I test this code:
 - Clean the solution.
 - Verify that .dlls are removed. GitExtUtils.dll is still remains though.
 - Start debugging GitExtensions
 - Verify that the AppVeyor plugin is loaded

Has been tested on (remove any that don't apply):
 - Windows 10
